app/models/organization_webhook.rb

```diff
@@ -9,7 +11,62 @@ class OrganizationWebhook < ApplicationRecord
   validates :github_organization_id, presence:   true
   validates :github_organization_id, uniqueness: true
 
-  def github_organization
-    @github_organization ||= organizations&.first&.github_organization
+  # External: Finds a User's token that has the `admin:org_hook` scope
+  # for creating the organization webhook.
+  #
+  # Example:
+  #
+  #   admin_org_hook_scoped_github_client
+  #   # => {token_string}
+  #
+  # Returns a User's token or raises a NoValidTokenError if one could not be found.
+  #
+  # Warning: This could potentially take very long for organizations
+  # of a large size, so invoke cautiously.
+  def admin_org_hook_scoped_github_client
+    token = if Rails.env.test?
+              users.first&.token
+            else
+              user_with_admin_org_hook_scope.sample&.token
```

Comment:
Should this be calling the method underneath? There's a small typo
```suggestion
              users_with_admin_org_hook_scope.sample&.token
```